gflownet/envs/tree.py

```diff
@@ -548,40 +548,44 @@ def set_state(self, state: List, done: Optional[bool] = False):
             )
         return super().set_state(state, done)
 
-    def sample_actions(
+    def sample_actions_batch(
```

Comment:
This is wrong, but I fixed it in https://github.com/alexhernandezgarcia/gflownet/pull/194 (tree had a separate `sample_actions_continuous` after latest PR).